.devcontainer/devcontainer.json

```diff
@@ -3,6 +3,7 @@
     "context": "..",
     "dockerfile": "Dockerfile"
   },
+  "initializeCommand": "cp .env.dist .env",
```

Comment:
Doesn't this obliterate any local changes?

Reply:
Yup good point -- I'll update that to make it not do that

Reply:
learned about a flag I didn't know about in the `cp` command... `-n` for no clobber :) tested locally and looks good just waiting for pre-commit checks

Reply:
fixed/pushed